DOCUMENTATION.md

```diff
@@ -0,0 +1,150 @@
+
+<br>
+
+PR creation is set to immediate meaning as soon as a branch with the updated dependencie is created so is the PR. It is recommended you do not override this value. You can set this to either create a PR upon successful repo status checks or passiing tests but this can be brittle and undermine the point of automatic dependency management. There is also a manual setting to require human approval, but this will create a bottleneck in your PR process. Note this means approval to create a PR, not like a code approval to merge an already existing one! Adding an additional step that may not be useful.
+
+<br>
+
+Automatically filters for our preset of 7 days for Stability Days. This is the number of days since the patch/update was released. This isn't an immediate green light that the patch will work for you though! You still have to look into the changes, test your code, cover all aspects of due diligence, as well as weigh the success of a patch/update by how many people have already merged it with no issue.
+
+[Back to top](#renovate-configuration)
+#
+<h2 align="center">
+Anatomy of a PR   
+</h2>
+   
+- ### Header will include stats on Stability Days or the number of days a release has been available as well as the percentage of other projects using that package that have successfully onboarded that release.
+    ![header](images/header.png)
+
+- ### Relevant CVE information to determine how critical a patch/update will be to your project.
+    ![cve](images/cve_info.png)
+
+- ### Links to release notes regarding the update. This allows you to read about possible breaking changes, and understand what was fixed. If you have to update your code to accomodate your code this is invaluable information to have.   
+    ![release_notes](images/patch_release_notes.png)
+
+- ### Current configuration settings that apply to this PR
+    ![config](images/configuration.png)
+
+- ### And a handy automatic rebase option if you need it if time passes while you're researching your update. This way you can keep your PR nice and fresh while you're preparing any code changes on the branch to accomodate the upgrade. Keep in mind this rebase will also happen as soon as a PR hits the stale date of 21 days! So rebase as often as you can if an update is requireing a little more rework than planned!
+    ![rebase](images/rebase_option.png)
+
+[Back to top](#renovate-configuration)
+#
+
+<h2 align="center">
+Language specific tips and tricks   
+</h2>
+
+### Python:
+- Renovate supports `pip`, `pipenv`, `setup.py`, and `setup.cfg` by default. You don't have to change anything if you are using any of these.
+- Renovate will then look up dependencies on `PyPi` and determine if upgrades are available.
+- The default tracking file Renovate looks for is `requirements.txt` if you're using different naming conventions you can add this to your local config. You can add file names or use regex to filter:
```

Comment:
Now that we've update the default `pip_requirements`, it matches more.  If you want to hoist this out of the default config, can you create the update PRs for the Notify projects that'll need it.

Reply:
Updated.